test/command/suite/select/filter/ordered_near_phrase/max_element_intervals/ngram.test

```diff
@@ -0,0 +1,22 @@
+table_create Entries TABLE_NO_KEY
+column_create Entries content COLUMN_SCALAR Text
+
+table_create Terms TABLE_PAT_KEY ShortText \
+  --default_tokenizer 'TokenNgram("unify_alphabet", false, \
+                                  "unify_digit", false)' \
+  --normalizer NormalizerNFKC121
+column_create Terms entries_content COLUMN_INDEX|WITH_POSITION Entries content
+
+load --table Entries
+[
+{"content": "abcXYZdef"},
+{"content": "abebcdXYZdef"},
+{"content": "abcdef"},
+{"content": "defXYZabc"},
+{"content": "XYZabc"},
+{"content": "abc123456789def"},
+{"content": "abc12345678def"},
+{"content": "abc1de2def"}
```

Comment:
Can we use more meaningful test data?
For example, if we focus on the max interval `3`, we should use border data such as interval `2`, `3` and `4` data.

Reply:
Fixed.